components/ui/infopopover.tsx

```diff
@@ -6,21 +6,15 @@ import {
 import { Info } from "lucide-react";
 
 type InfoProps = {
-  iconSize: number;
   content: any;
   iconStyle?: string;
 };
 
-export function InfoPopover({ iconSize, content, iconStyle }: InfoProps) {
+export function InfoPopover({ content, iconStyle }: InfoProps) {
   return (
     <Popover>
       <PopoverTrigger asChild>
```

Comment:
`<Popover>
      <PopoverTrigger asChild>
        <Button variant="outline" size="icon" className={iconStyle}>
          <Info size={iconSize} role="img" aria-label="Info" />
        </Button>
      </PopoverTrigger>
      <PopoverContent className="w-80">
        <div className="grid gap-4">{content}</div>
      </PopoverContent>
    </Popover>`
    
    Seems to work for the clickable area

Reply:
I just saw this after pushing my changes! 😮 I was able to make the background clickable by playing around with the styling and removing the Button wrapper.